.gitignore

```diff
@@ -2,6 +2,7 @@
 .codeintel
 .coverage
 .DS_Store
+.idea
```

Comment:
I prefer to keep these developer's choice ignores out of here, and in a global gitignore instead. Otherwise we'll end up with an ever growing list.

The .codeintel and sublime ones snuck in at some point!